storybookjs/storybook
# support .vue extention resolving

Issue:
In Vue projects, Cannot import single file components (.vue) without extension.

## What I did
add `.vue` extention to `resolve.extensions` of webpack config.

## How to test

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

~~If your answer is yes to any of these, please make sure to include it in your PR.~~
